Application/mobile/app_code/lib/pages/container/container_list.dart

```diff
@@ -57,32 +79,36 @@ class ContainerCard extends StatelessWidget {
             MaterialPageRoute(
                 // id must be the same from web to mobile
                 builder: (context) => ContainerDetailsPage(
-                    containerId: container.id.toString())));
+                    containerId: container.id)));
       },
       child: Container(
         height: 120,
-        margin: const EdgeInsets.only(right: 25.0, left: 25.0, top: 10.0),
-        // Adjust the padding here
-        child: Card(
-          elevation: 5,
-          shadowColor: Colors.blueAccent,
-          shape: RoundedRectangleBorder(
-            borderRadius: BorderRadius.circular(10.0),
-          ),
-          child: Column(
-            children: [
-              ListTile(
-                title: Text(container.id.toString()),
-                // subtitle: Text(container.price.toString()),
-                leading: const Row(
-                  mainAxisSize: MainAxisSize.min,
-                  children: [
-                    SizedBox(width: 10),
-                    Text("name"),
-                  ],
+        margin: EdgeInsets.only(
+            right: 25.0, left: 25.0, top: 10.0),
+        child: Container(
+          child: Card(
+            elevation: 5,
+            shadowColor: context.select((ThemeProvider themeProvider) => themeProvider.currentTheme.dividerColor),
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(10.0),
+            ),
+            child: Column(
+              children: [
+                ListTile(
+                  title: container.city != null ? Text(container.city!) : null,
+                  subtitle: container.address != null ? Text(container.address!) : null,
+                  leading: Row(
+                    mainAxisSize: MainAxisSize.min,
+                    children: [
+                      SizedBox(
+                        width: 10
+                      ),
+                      Text("prix : " + container.price.toString()),
```

Comment:
Uppercase and precise which currency

Reply:
What do you mean ?

Reply:
The 'P' of "Prix"
And don't forget to format the file